code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -172,6 +172,10 @@
 			playsound(L, 'sound/weapons/genhit.ogg', 50, 1)
 			return
 
+	if(iscarbon(L))
```

Comment:
Now this shouldn't need to be anything to consider as shock_internal organs parses through the list of internal organs...BUT AS THIS IS BYOND CODE...would adding an hasorgans check here be safe?